episodes/03-hello-open-source.md

```diff
@@ -33,6 +227,8 @@ exercises: 20
 - Markdown is a a style of markup that is human and machine readable, it is rendered as HTML
 - HTML is the formt of markup that web browsers use. 
 - Source content is what we edit and may content developer informtion, rendered content (or built) is what is for the reader, output by a build process
+- GitHub Pages offer an automated way to create a website that is version controlled and accessible for collaboration
+- Collaborating on a GitHub Pages website uses the same Git/GitHub workflow you learned for collaborating via a GitHub repository
```

Comment:
2nd one here is not learned yet... so rephrase to futre

1st is tht it offer free hosting 

also line above fix typo in information